setup.py

```diff
@@ -3,30 +3,7 @@
 from setuptools import setup
 from setuptools.command.test import test as TestCommand
 
-
-class PyTest(TestCommand):
-    def finalize_options(self):
-        TestCommand.finalize_options(self)
-        self.verbose = True
-
-    def run_tests(self):
-        import pytest
-        errno = pytest.main(self.test_args)
-        sys.exit(errno)
-
-
-def extract_version(module='utide'):
```

Comment:
Most of the PR is bolierplate added automatically by `versioneer`. We need to do that only once.
I removed this function b/c it is obsolete now.